qtpy/tests/test_qtdesigner.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+
+import pytest
+from qtpy import PYSIDE2, PYSIDE
+
+@pytest.mark.skipif(PYSIDE, reason="QtDesigner is not avalaible in PySide")
+@pytest.mark.skipif(PYSIDE2, reason="QtDesigner is not avalaible in PySide2")
```

Comment:
`@pytest.mark.skipif(PYSIDE2 or PYSIDE, reason="QtDesigner is not avalaible in PySide2")`

?